tests/ctags/Makefile.am

```diff
@@ -147,6 +147,9 @@ test_sources = \
 	cxx11enum.cpp					\
 	cxx11-final.cpp					\
 	cxx11-noexcept.cpp				\
+	gdscript-inner-class.gd			\
+	gdscript-modifiers.gd			\
+	gdscript-no-implicit-class.gd	\
```

Comment:
Just a nitpicky comment: better to sort the files alphabetically.

Reply:
I think it goes hijklmn, so these are already in alphabetical order?

Reply:
Oh wait somehow I inserted these in a different place on the list than I thought I did, adjusting